Vagrantfile

```diff
@@ -30,9 +30,6 @@ sudo apt-get update -qq -y
 
 sudo apt-get install -qq curl php5 git libspreadsheet-writeexcel-perl libdbd-sqlite3-perl
 
-curl -sS https://getcomposer.org/installer | php
```

Comment:
it seem like that this removal in fact results in a non working installation now. even in the checked out branch I have the composer file there (1,8MB), but it is not found by the vagrant provisioning.